redisson/src/main/java/org/redisson/connection/SentinelConnectionManager.java

```diff
@@ -212,12 +212,10 @@ private void checkAuth(SentinelServersConfig cfg) {
             try {
                 RedisConnection c = client.connect();
                 connected = true;
-                try {
-                    c.sync(RedisCommands.PING);
-                } catch (RedisAuthRequiredException e) {
-                    usePassword = true;
-                }
+                c.sync(RedisCommands.PING);
                 break;
+            } catch (RedisAuthRequiredException e) {
```

Comment:
could you clarify how does this change help?

Reply:
@mrniko : I have updated the code, if in auth check we get RedisConnectionException then the service will retry to establish the connection with the password provided in config.

Reply:
Let me recheck this behaviour. Because it shoud throw RedisAuthRequiredException directly and not wrapped by RedisConnectionException

Reply:
@mrniko : I checked the code was throwing RedisConnectionException not RedisAuthRequiredException. Also, we were running Redis with a single sentinel in our lower environment, we needed to retry connecting the Sentinel with auth from the beginning. I have made the changes, please review it.